crates/bitcoin-da/src/service.rs

```diff
@@ -265,6 +272,18 @@ impl BitcoinService {
         }
     }
 
+    /// Queue and try sending transaction to DA
+    pub async fn send_transaction_with_fee_rate(
+        &self,
+        tx_request: DaTxRequest,
+        fee_sat_per_vbyte: u64,
+    ) -> Result<Vec<Txid>> {
+        self.queue_da_transaction_with_fee_rate(tx_request, fee_sat_per_vbyte)
+            .await?;
+
+        self.process_transaction_queue().await
```

Comment:
Also I don't like this one. What's the point of a queue if the queue is processed only after queue_da_transaction_with_fee_rate. So in theory it will be filled, read, and empty after send_transaction_with_fee_rate call is over. So... it's useless?

Reply:
Ah, I see. Thanks